src/ddmd/backend/cod3.c

```diff
@@ -2479,7 +2479,7 @@ void genmovreg(CodeBuilder& cdb,unsigned to,unsigned from)
 {
 #if DEBUG
         if (to > ES || from > ES)
-                printf("genmovreg(c = %p, to = %d, from = %d)\n",c,to,from);
+                printf("genmovreg(c = %p, to = %d, from = %d)\n",cdb,to,from);
```

Comment:
I guess it should be `&cdb`. Please change the format string, too.

Reply:
I just changed it to:
```
                printf("genmovreg(to = %d, from = %d)\n",to,from);
```